test/extended/main.go

```diff
@@ -0,0 +1,12 @@
+package extended
+
+import (
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+)
+
+var _ = g.Describe("[Jira:OAS][sig-api-machinery] sanity test", func() {
```

Comment:
```suggestion
var _ = g.Describe("[Jira:openshift-apiserver][sig-api-machinery] sanity test", func() {
```

Lets use exact component name